wizard_builder/models.py

```diff
@@ -3,21 +3,30 @@
 from polymorphic.models import PolymorphicModel
 
 from django import forms
-from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.sites.models import Site
+from django.core.exceptions import ImproperlyConfigured, ObjectDoesNotExist
 from django.db import models
+from django.db.utils import OperationalError
 from django.utils.safestring import mark_safe
 
+from .managers import PageBaseManager
+
 
 def get_page_position():
     try:
         return PageBase.objects.latest('position').position + 1
-    except ObjectDoesNotExist:
+    except (ObjectDoesNotExist, OperationalError):
```

Comment:
the default attribute gets evaluated in migration 1, but uses the most recent version of the model which isn't valid until migration 2. So you get a `sql...OperationalError` because of trying to access an attribute that doesn't exist yet

coincidentally I ran into this same issue in https://github.com/SexualHealthInnovations/django-wizard-builder/pull/25, albeit for very different reasons